src/main/java/com/twilio/oai/api/ApiResourceBuilder.java

```diff
@@ -58,6 +63,11 @@ public ApiResourceBuilder updateOperations(ISchemaResolver<CodegenParameter> cod
 
             requiredPathParams.addAll(codegenOperation.pathParams);
             codegenOperation.vendorExtensions = mapOperation(codegenOperation);
+
+            codegenOperation.vendorExtensions.put("hasRequiredFormParams",
```

Comment:
these should be added under mapOperation(codegenOperation). Let's have one place where all codegenOperation vendorExtensions are added.